test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -95,10 +102,14 @@ public ServerContext build() {
         builder.put("--Prefix", prefix);
 
         // Provide the server with event loop groups
-        if (implementation.equals("local")) {
+        if (implementation.equals("local") && isTest()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651477)